src/redux/watchList/reducer.js

```diff
@@ -0,0 +1,61 @@
+import { FETCH_WATCHLIST_SUCCESS, POST_WATCHLIST_SUCCESS } from './actionCreator';
+
+const watchlist1 = {
+  id: 1,
+  name: 'A share',
+  stocks: [],
+};
+
+const watchlist2 = {
+  id: 2,
+  name: 'NSDQ',
+  stocks: [],
+};
+
+const initalState = {
+  watchlists: [watchlist1, watchlist2],
```

Comment:
the whole state could be a simple array, so you can avoid the `useSelector(state => state.watchlists.watchlists)` mappings.